amqp/src/main/scala/akka/stream/alpakka/amqp/scaladsl/AmqpFlow.scala

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.amqp.scaladsl
+
+import akka.Done
+import akka.stream.alpakka.amqp._
+import akka.stream.scaladsl.Flow
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
+object AmqpFlow {
+
+  /**
+   * Creates an [[AmqpFlow]] that accepts [[WriteMessage]] elements and emits [[WriteResult]].
```

Comment:
Scaladoc requires fully-qualified class names for everything that is not package local.
You can remove the links and just put the classes in backticks -- the types are referenced in the type signature anyway.

Reply:
Done.